assets/javascripts/discourse/controllers/admin-plugins-patreon.js

```diff
@@ -93,8 +94,13 @@ export default Controller.extend({
 
         this.set("updatingData", false);
 
-        bootbox.alert(I18n.t("patreon.refresh_page"), () => {
-          window.location.pathname = getURL("/admin/plugins/patreon");
+        this.dialog.alert({
+          message: I18n.t("patreon.refresh_page"),
+          didConfirm: () => {
```

Comment:
I believe the previous behaviour was to redirect to `/admin/plugins/patreon` when cancelling as well. In the new service, that's a separate action, under `didCancel`, so we should add that as well.

Reply:
Yes, I'll change that